lib/GADS.pm

```diff
@@ -1660,7 +1659,7 @@ post '/print' => require_login sub {
     );
 };
 
-prefix '/:layout_name' => sub {
+prefix '/:layout_name' => sub {data_calendar
 
```

Comment:
Thanks for the PR @stolman-digitpaint - I'm just checking whether the above ```data_calendar``` is meant to be there? It doesn't look quite right.

Reply:
it's not supposed to be there, no. I've removed this issue earlier, but there was a merge conflict when i pushed some finalizing local changes, and in the resulting merge they seem to have come back. i'll fix it shortly

Reply:
issue is resolved